app_bikes/providers/star.py

```diff
@@ -0,0 +1,57 @@
+# coding: utf-8
+
+import requests
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _L
+
+from . import base
+from .. import settings
+
+
+class StarProvider(base.BaseProvider):
+    STAR_API_BASE_URL = 'https://data.explore.star.fr/api/records/1.0/search'
+
+    verbose_name = _L(u'Star - Rennes')
+
+    @classmethod
+    def get_stations(cls, query):
+        if query.isdigit():
+            query = 'nom:{query} OR idstation:{query}'.format(query=query)
+        else:
+            # As idstation is an int in the API it raises an error otherwise
```

Comment:
if the query is a string, the call will raise an error because idstation is store as an int in the webservice database, so we have to remove this field